src/docs/product/alerts-notifications/index.mdx

```diff
@@ -8,19 +8,40 @@ description: "Sentry categorizes notifications into alerts and other types of no
 
 Sentry categorizes notifications broadly into alerts and non-alert notifications.
 
-## Alerts
+## Error Alerting
 
-[Alerts](/product/alerts-notifications/alerts/) notify you about problems in your application and can be sent to many supported integrations. You can set alerts on both errors and performance data.
+Alerts notify you about problems in your application and can be sent to many supported integrations. You can set alerts on both errors and performance data.
```

Comment:
Can we be a bit more explicit here? Perhaps "Alerts let you know of problems occurring in your application, and can be sent via email, chatops, and many other channels depending on the integrations you've created using our [Integration Platform](link). 

The last sentence here is confusing given that we're in as section entitled "Error Alerts"